README.md

```diff
@@ -0,0 +1,162 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Home Page
+- About page
+<!-- - Contact Me section -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Demo <a name="#video"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to have [VS code](https://code.visualstudio.com), the [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) extension and a web browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```shell
+  cd my-folder
+  git clone git@github.com:Jeddaa/microverse-project1
+```
+
+### Install
+
+You don't have to install the project. All you need to do is clone it with the above command
+
+<!-- Install this project with: -->
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+Once you have the live server extension, you need the click on the **Go Live** icon at the bottom of the VS code window or you could hit **'Alt + L + O'** on your keyboard.
+
+### Run tests
+
+To run tests, run the following command:
+
+```shell
+  npx hint .
+  npx stylelint "\*_/_.{css,scss}"
+  npx eslint .
+
+```
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [Jeddaa](https://github.com/jeddaa)
+- Twitter: [@\_jeddaah](https://twitter.com/_jeddaah)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- Contributions have been made and accepted by: -->
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Kindly help check for errors or bugs. Feel free to contact me on what could be done better
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+A special thank you to the author of the original design idea Cindy Shin in [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), my learning partners and standup team for their help and support. Thank you to my friends for always answering my numerous questions too.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Jeddaa/microverse-project1/blob/main/LICENSE) licensed.
```

Comment:
- [ ] It would be better if we add our name to the license file so that it represents our project and helps make our README professional